multiversx_sdk_core/testutils/wallets.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+from typing import Dict
+
+from multiversx_sdk_wallet import UserPEM
+
+alice_pem = "multiversx_sdk_core/testutils/testwallets/alice.pem"
```

Comment:
Can also be `Path(__filename__).parent / testwallets / alice.pem` or something like that.

Reply:
Changed to `Path(__file__).parent / testwallets / alice.pem`